2_drivers.yml

```diff
@@ -17,13 +17,39 @@ targets:
   #out_data/placeholder_for_gefs_archive:
    # command: copy_files()
     
-  #out_data/palceholder_for_nldas:
-   # command: copy_files()
-   
+  # NLDAS Files
+  ## NLDAS_time[0.377900]_x[397]_y[136].csv
+  out_data/NLDAS_time[0.377900]_x[397]_y[136].csv:
+    command: file.copy(from = 'in_data/NLDAS_time[0.377900]_x[397]_y[136].csv',
+    to = target_name)
+  ## NLDAS_time[0.377900]_x[401]_y[136].csv
+  out_data/NLDAS_time[0.377900]_x[401]_y[136].csv:
+    command: file.copy(from = 'in_data/NLDAS_time[0.377900]_x[401]_y[136].csv',
```

Comment:
Just hopping in here to point out a gotcha - 

`file.copy` has an `overwrite` arg that is `FALSE` by default, meaning that even if `in_data/NLDAS_time[0.377900]_x[401]_y[136].csv` has changed but the `out_data/NLDAS_time[0.377900]_x[401]_y[136].csv` was already created based on the prior update, it won't get updated with the new change.

Reply:
ah good to know thanks!